src/components/Theme/README.md

```diff
@@ -723,20 +724,48 @@ The Theme component makes these reusable CSS variables available to all DOM node
 
 ### Colors
 
+#### Neutral colors
+
 Variable | Purpose
 -|-
---maker-color-neutral-0 | neutral-0
+--maker-color-neutral-0 | neutral-0 (min contrast against background)
 --maker-color-neutral-10 | neutral-10
 --maker-color-neutral-20 | neutral-20
 --maker-color-neutral-80 | neutral-80
 --maker-color-neutral-90 | neutral-80
---maker-color-neutral-100 | neutral-100
+--maker-color-neutral-100 | neutral-100 (max contrast against background)
+
+#### General colors
+
+Variable | Purpose
+-|-
 --maker-color-primary | primary color
 --maker-color-background | background color
---maker-color-heading | heading color
+--maker-color-heading | heading text color
 --maker-color-body | body text color
---maker-color-elevation | elevation color
---maker-color-overlay | overlay color
+--maker-color-elevation | elevation color, e.g. color of elevated elements, like ActionBarButtons or selected Segments in SegmentedControl
+--maker-color-overlay | overlay color, e.g. color of Modal layer, Dialog layer, etc
+
+#### Contextual colors
+
+Variable | Purpose
+-|-
+--maker-color-primary-fill | primary fill color, same as primary color
```

Comment:
I'm not sure if i want to make contextual colors publicly available yet. The logic for how they're applied to Maker components and how they react over background is not super intuitive and I would prefer to just make add success or critical variants for a component that doesn't currently support those patterns than have engineers try to make sense of our generated contextual colors.